api.planx.uk/gis/digitalLand.js

```diff
@@ -185,7 +190,23 @@ async function go(localAuthority, geom, extras) {
     }
   }
 
-  return { url: url, constraints: formattedResult };
+  // --- METADATA ---
+  // additionally fetch metadata from Digital Land's "dataset" endpoint for extra context
+  let metadata = {};
+  const urls = activeDatasets.map((dataset) => `https://www.planning.data.gov.uk/dataset/${dataset}.json`);
+  await Promise.all(urls.map(url => 
```

Comment:
Two thoughts: 
- yes, all requests could be wrapped in the `Promise.all()` to run concurrently, but actually, we probably shouldn't proceed with fetching metadata at all if our constraints request throws an error? I'm not doing this currenty, but will adjust
- caching makes sense here (although unfortunately when profiling this component, I think `/roads` is still by far the slowest culprit because of the USRN string matching even with 10+ new metadata requests in play). is it okay if i similarly punt introducing this dependency until I convert to ts? planning to pick that up this week still once Ian is set on styling front